sdk/storage/azure-storage-data-movement/src/main/java/com/azure/storage/data/movement/Temp.java

```diff
@@ -4,7 +4,6 @@
 package com.azure.storage.data.movement;
 
 /**
- * Temporary Class
+ * Placeholder class
  */
-public class Temp {
-}
+public class Temp { }
```

Comment:
what is this for?

Reply:
module-info didn't like the package being empty, so I put the Temp class back for until something else ends up in the main package.